mypy.ini

```diff
@@ -0,0 +1,2 @@
+[mypy]
+ignore_missing_imports = True
```

Comment:
Is there any particular reason to add this?

Reply:
some package required don't have stub file which needed by mypy. by default it will raise error, so i exclude it